Installation/changes.html

```diff
@@ -187,6 +187,30 @@ <h2 id="release4.12">Release 4.12 </h2>
   other types of input than the RANSAC output.</li>
 </ul>
 <!-- Spatial Searching and Sorting -->
+<h3>dD Spatial Searching</h3>
+<ul>
+  <li>
+  Improved the performance of the kd-tree in some cases:<br />
+  1. Not storing the points coordinates inside the tree usually 
+      generates a lot of cache misses, leading to non-optimal 
+      performance. This is the case when indices are stored inside 
+      the tree, or when the points coordinates are stored in a
+      dynamically allocated array (e.g., <code>CGAL::Epick_d</code>
+      with dynamic dimension).
+      In this case, the new EnablePointsCache template parameter of the
+      <code>CGAL::Kd_tree</code> class can be set to 
+      <code>CGAL::Tag_true</code>. The points coordinates
+      will then be cached in an optimal way. This will increase memory
+      consumption but provides better search performance. See the updated
+      <code>GeneralDistance</code> and <code>FuzzyQueryItem</code>
+      concepts for additional requirements when using such a cache.
+  2. In most cases (e.g., Euclidean distance), the distance computation
+      algorithm knows before its end that the distance will be greater
+      than or equal to some given value. This is used in the (orthogonal)
+      k-NN search to interrupt some distance computations before its end,
+      saving precious milliseconds, in particular in medium-to-high dimension.
+  </li>
```

Comment:
This text will not be correctly formatted, as a HTML text.

Instead of `1./2.`, you must use the `<ol>` and `<li`> tags, to get a HTML ordered list.

Reply:
Of course you're right. Fixed in 35563ebe73cd1f6bca1b6ed788b495d7a1e4bfd9.